connectors/v2/anghami.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+
+Connector.artistSelector = '.track-artist';
+
+Connector.trackSelector = '.track-title';
+
+Connector.currentTimeSelector = '.position';
+
+Connector.durationSelector = '.duration';
+
+Connector.playButtonSelector = '.play-pause-button paused';
```

Comment:
Well, 'play button' is a button that means to be invisible when the song is playing. Moreover, the `.play-pause-button paused` selector is not valid. Scrobbling works fine in this case, the extension scrobbles paused songs too.

Also, '.play-pause-button' element has a class named 'paused' only if song is *paused*, so it's enough to check if 'paused' class is present, like this way:
```js
Connector.isPlaying = function() {
	return !$('.play-pause-button').hasClass('paused');
};
```